src/cli.js

```diff
@@ -32,7 +34,7 @@ let cli = {
       // Show help
       console.log(options.generateHelp());
     } else {
-      nibbler.setExtensions(extensions);
+      nibbler.setExtensions(extensions, config);
```

Comment:
It seemed that not passing a config appeared to still work (I'm assuming `CLIEngine` with a undefined config just used the default), but I felt dirty doing it.

Renaming the function and passing different objects in sounds like a much better solution so I'll look at doing that.